packages/rebber/__tests__/fixtures/inline-code.expected.tex

```diff
@@ -0,0 +1,9 @@
+a paragraph \texttt{with inline code}.
+
+\texttt{a
+multiline
+inlinecode}
+
+\texttt{a code with \textbackslash}
```

Comment:
This should be `\textbackslash{}\textbackslash{}` according to the markdown fixture (`\\`)